src/openforms/registrations/contrib/objects_api/tests/test_backend_v2.py

```diff
@@ -376,6 +376,33 @@ def test_hidden_component(self):
         record_data = object_data["record"]["data"]
         self.assertEqual(record_data["textfield"], "some_string")
 
+    def test_public_reference_available(self):
+        submission = SubmissionFactory.create(
+            with_public_registration_reference=True,
+            public_registration_reference="OF-911",
+            auth_info__plugin="irrelevant",
+            auth_info__is_eh_bewindvoering=True,
+        )
+        ObjectsAPIRegistrationData.objects.create(submission=submission)
+        v2_options: RegistrationOptionsV2 = {
+            "version": 2,
+            # "objects_api_group": self.objects_api_group.pk,
```

Comment:
@Viicos these are because of your PR - ideally yours can get merged first and then I can rebase these tests

Reply:
Sounds good. Isn't the `objects_api_group` available on `master`?

Reply:
no, your PR updates the `setUpTestData`